install/linux/usr/share/odemis/sim/enzel-sim-xt.odm.yaml

```diff
@@ -6,7 +6,7 @@ ENZEL-Sim: {
         "Sample Thermostat", "Cooler Controller",
         "SEM E-beam Full", "SEM Detector Full",
         "SEM Detector", "EBeam Control",
-        "EBeam Focus", "EBeam Detector",
+        "EBeam Focus", "FIB ETD",
```

Comment:
ETD? What do that stand for?

Reply:
That's another name for "secondary electron detector". It stands for Everhart–Thornley detector. Maybe I could rename it to "FIB Detector" or "FIB SE Detector". Any preference or suggestion?